nf_core/pipeline-template/.github/workflows/ci.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         NXF_VER:
           - "24.04.2"
           - "latest-everything"
+        engine:
```

Comment:
```suggestion
        profile:
```

Reply:
I'd rather keep profile for the different test profile, but happy to change engine for anything else